travel_journey/report/travel_journey_webkit.py

```diff
@@ -41,19 +41,30 @@ def _get_signer(self):
             return ''
 
     def _get_passenger(self, journey):
+        user_object = self.pool['res.users']
+        employee_object = self.pool['hr.employee']
         try:
             passenger_name = journey.passenger_id.partner_id.name or ''
+            # Get job name for passenger
+            partner_id = journey.passenger_id.partner_id.id
+            user_ids = user_object.search(
+                self.cr, self.uid, [('partner_id', '=', partner_id)])
+            employee_ids = employee_object.search(
+                self.cr, self.uid, [('user_id', 'in', user_ids)])
+            job_name = employee_object.browse(
+                self.cr, self.uid, employee_ids[0]).job_id.name or ''
```

Comment:
One last question: employee_ids[0]).job_id can be empty? If does, you should protect the expression.